src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -708,8 +739,20 @@ private void AnalyzeString(string strValue, Statistics stat)
         /// </summary>
         private void AnalyzeNumber(double numValue, Statistics stat)
         {
-            if (_measureSum || _measureAverage)
+            if (_measureSum || _measureAverage || _measureStdDeviation)
+            {
+                stat.sumPrevious = stat.sum;
                 stat.sum += numValue;
+            }
+            if (_measureStdDeviation)
+            {
+                double countMinusOne = stat.count - 1;
+                double avgPrevious = (countMinusOne == 0) ? 0 : (stat.sumPrevious / countMinusOne);
+                double avg = stat.sum / stat.count;
+                stat.stdDeviation *= countMinusOne;
+                stat.stdDeviation += (numValue - avgPrevious) * (numValue - avg);
+                stat.stdDeviation /= stat.count;
+            }
```

Comment:
I think the implementation is slightly incorrect: if we have only 1 entry you algorithm doesn't output 0, like it should. I also expected to see a `(count - 2) / (count - 1)` but you have an equivalent of `(count - 1) / (count)`. I think that's the problem - one-off error.

Also, please address @iSazonov 's comments: remove the line with empty tag and  revert all the changes in `MeasureObjectStrings.resx` since we don't use this string for a custom error message anymore.